qml/EventUtils.qml

```diff
@@ -78,9 +78,32 @@ QtObject{
             // E.g. "Daily; until 12/12/2014."
             str = i18n.tr("%1; until %2").arg(recurrence).arg(rule.limit.toLocaleString(Qt.locale(), dateFormat))
         }
+
+        if (rule.interval !== undefined && rule.interval > 1) {
+            str = str + getIntervalString(rule);
+        }
+
         return str;
     }
 
+    function getIntervalString(rule) {
+        var index = rule.frequency;
+
+        if (index == RecurrenceRule.Daily) {
+            return i18n.tr("; every %1 days").arg(rule.interval);
```

Comment:
It would be nice to support plurals here: https://docs.ubuntu.com/phone/en/apps/api-qml-current/Ubuntu.Components.i18n#tr-method-2

Reply:
Does that really matter here? since in this case its always plural...